docs/test-pr-guide.md

```diff
@@ -14,6 +14,7 @@
 
 ```
 git clone https://github.com/<contributor>/mentorship-backend/
+git fetch upstream pull/ID/head:<branch_name>
```

Comment:
Would be better if we can explain what ID stands for here :thinking:

Reply:
ID stands for

Reply:
First of all, no need to use sir.